### ISSUE TITLE
Operator with engine CUDNN is not available for operator <*>

### ISSUE THREAD

**Issue by khaerulumam42** [shinseung428/Detectron_webcam_realtime#1]
Unfortunately I got error like this 
`I0308 13:33:15.501793 17393 operator.cc:173] Operator with engine CUDNN is not available for operator AffineChannel.
I0308 13:33:15.502348 17393 operator.cc:173] Operator with engine CUDNN is not available for operator AffineChannel.
I0308 13:33:15.502640 17393 operator.cc:173] Operator with engine CUDNN is not available for operator AffineChannel.
I0308 13:33:15.502938 17393 operator.cc:173] Operator with engine CUDNN is not available for operator AffineChannel.
I0308 13:33:15.503180 17393 operator.cc:173] Operator with engine CUDNN is not available for operator AffineChannel.
I0308 13:33:15.503214 17393 operator.cc:173] Operator with engine CUDNN is not available for operator Sum.`

This error also found when I run proper script from detectron

Any idea?

**Comment by useafterfree** [comment]
Do you have CUDNN installed?

**Comment by khaerulumam42** [comment]
Yes, I have installed CUDNn, i have checked using `cat /usr/local/cuda/include/cudnn.h | grep CUDNN_MAJOR -A 2` and got this 

> #define CUDNN_MAJOR      6
#define CUDNN_MINOR      0
#define CUDNN_PATCHLEVEL 21
--
#define CUDNN_VERSION    (CUDNN_MAJOR * 1000 + CUDNN_MINOR * 100 + CUDNN_PATCHLEVEL)

#include "driver_types.h"
